gengen/README.md

```diff
@@ -0,0 +1,37 @@
+# gengen
+
+gengen is a filecoin tool that builds the genesis.car, which is used to seed the
+genesis block.
+
+### Example
```

Comment:
non-blocking: in addition to this example it would be nice if we provided examples of:
- How to initialize a filecoin node with the `genesis.car` file
- How to use the `*.key` artifacts of the gengen call
- What the `setup.json` file does -- e.g. it allocates power and funds to an address that is in the genesis block.